cmd/habitat/community/consensus/cluster/cluster.go

```diff
@@ -35,8 +36,8 @@ type ClusterManager struct {
 	raftClusterService *raft.ClusterService
 }
 
-func NewClusterManager() *ClusterManager {
-	raft := raft.NewClusterService()
+func NewClusterManager(host host.Host) *ClusterManager {
```

Comment:
why does this take in a host instead of create one? will we share a libp2p host for different services?

Reply:
I think other services will eventually use it, but nothing in the super near term. The main possible use is if we wanted apps to have a RPC system between themselves